### PR TITLE
Fixed frontend formatting

### PR DESCRIPTION
**Pull request by carmenBen**
Fixed legend formatting to make it standout less.
Fixed sizing of panels to fit on page.
Fixed formatting to add space after countries in selected dropdown.
Made extrema countries clickable and modal pops up.

Note: Accidentally requested new review from Haylee after she approved this PR already.

### CODE REVIEW

File: search-sentiments-map/public/scripts/map-script.js [hunk 1]
```diff
@@ -155,24 +157,33 @@ function loadCountryData() {
...
       isSentimentMode ?
-          'Most Positive Country: ' + countryMax + 
-              ', Most Negative Country: ' + countryMin :
-          'Most Popular Country: ' + countryMax;
+          'Most Positive Country: <span class="modal-click" id="' +
+          countryMaxId + '">' + countryMaxName + 
+          '</span>, Most Negative Country: <span class="modal-click" id="' +
+          countryMinId + '">' + countryMinName + '</span>':
+          'Most Popular Country: <span class="modal-click" id="' +
+          countryMaxId + '">' + countryMaxName + '</span>' ;
```
Comment (hayleejane3) [inline search-sentiments-map/public/scripts/map-script.js:182]: Each item will have its own line here.

File: search-sentiments-map/public/scripts/map-script.js [hunk 2]
```diff
@@ -141,10 +141,10 @@ function loadCountryData() {
...
   let dataVariableMin = Number.MAX_VALUE;
   // Countries with the minimum and maximum scores for current topic. 
-  let countryMaxName = '';
-  let countryMaxId = '';
-  let countryMinName = '';
-  let countryMinId = '';
+  let maxScoreCountryName = '';
+  let maxScoreCountryId = '';
+  let minScoreCountryName = '';
+  let minScoreCountryId = '';
```
Comment (hayleejane3) [inline search-sentiments-map/public/scripts/map-script.js:147]: Alphabetical order so the max (Id then name) + min (id then name)

Comment (ntarn) [reply search-sentiments-map/public/scripts/map-script.js:147]: I think in a previous comment similar to this one, you said that we wanted the countryNames together and the countryId's together? or is it different for this case?

File: search-sentiments-map/public/scripts/modal-script.js [hunk 3]
```diff
@@ -27,16 +27,22 @@ function onClickRegion(e) {
...
+  // Update Modal with information for relevant country.
   document.getElementById('modal-title').innerText = countryName;
-  displayTopResultsForCurrentTrend(countryId);
-  setCountryTrends(countryId);
-  displaySentimentChartForCurrentTrend(countryId);
-  displayPopularityTimeline(countryId);
+  displayTopResultsForCurrentTrend(countryCode);
+  setCountryTrends(countryCode);
+  displaySentimentChartForCurrentTrend(countryCode);
+  displayPopularityTimeline(countryCode);
```
Comment (ntarn) [inline search-sentiments-map/public/scripts/modal-script.js:45]: When you merge master into this branch, we have the countryCode as a global varialbe, so update the parameter name here. Also make sure to update the global var countryCode when you open the modal from clicking on the most popular countries bc it's needed for all the updated methods in master

File: search-sentiments-map/public/scripts/modal-script.js [hunk 4]
```diff
@@ -27,16 +27,22 @@ function onClickRegion(e) {
...
     return;
   }
-  $('#region-info-modal').modal('show');
 
   // Update Modal with information for relevant country.
   const countryName = e.feature.getProperty('name');
   const countryId = e.feature.getId();
+  openModal(countryId, countryName);
+}
+
```
Comment (ntarn) [inline search-sentiments-map/public/scripts/modal-script.js:36]: Add Jsdoc comment

File: search-sentiments-map/public/style.css [hunk 5]
```diff
@@ -26,6 +26,14 @@ i {
   width: 360px;
 }
 
+.modal-click {
+	cursor: pointer;
```
Comment (ntarn) [inline search-sentiments-map/public/style.css:30]:
```suggestion
  cursor: pointer;

```

File: search-sentiments-map/public/scripts/map-script.js [hunk 6]
```diff
@@ -155,24 +157,36 @@ function loadCountryData() {
...
       isSentimentMode ?
-          'Most Positive Country: ' + countryMax + 
-              ', Most Negative Country: ' + countryMin :
-          'Most Popular Country: ' + countryMax;
+          'Most Positive Country: ' +
+          '<span class="modal-click" id="' + maxScoreCountryId + '">' +
+          maxScoreCountryName + '</span>' + 
+          ', Most Negative Country: ' +
+          '<span class="modal-click" id="' + minScoreCountryId + '">' +
+          minScoreCountryName + '</span>':
```
Comment (ChenyuHeidiZhang) [inline search-sentiments-map/public/scripts/map-script.js:182]:
```suggestion
          minScoreCountryName + '</span>' :
```

File: search-sentiments-map/public/scripts/map-script.js [hunk 7]
```diff
@@ -155,24 +157,36 @@ function loadCountryData() {
...
+          'Most Popular Country: ' +
+          '<span class="modal-click" id="' + maxScoreCountryId + '">' +
+          maxScoreCountryName + '</span>' ;
+
+  $(".modal-click").click(function(){
```
Comment (ChenyuHeidiZhang) [inline search-sentiments-map/public/scripts/map-script.js:187]:
```suggestion
  $('.modal-click').click(function() {
```

File: search-sentiments-map/public/scripts/modal-script.js [hunk 8]
```diff
@@ -27,16 +27,22 @@ function onClickRegion(e) {
...
+function openModal(countryCode, countryName) {
```
Comment (ChenyuHeidiZhang) [inline search-sentiments-map/public/scripts/modal-script.js:37]: One suggestion: Can we center the map to that country when opening the modal?

File: search-sentiments-map/public/style.css [hunk 9]
```diff
@@ -129,7 +143,8 @@ input:checked + .slider:before {
 
 #right-div {
   background-color: rgba(255, 255, 255, 0.5);
-  overflow: hidden;
+  height: 70%;
```
Comment (ChenyuHeidiZhang) [inline search-sentiments-map/public/style.css:146]: I personally prefer the height to wrap the content (as in overflow: hidden). If you see the 70% height on a monitor there would be a lot of white space below

Comment (carmenBen) [reply search-sentiments-map/public/style.css:146]: Okay, but wrapping the overflow means that on small screens the user can still view all of the content. I feel like extra whitespace is a smaller problem.

Basically in overflow hidden, on smaller laptop screen, the content cuts off complete. I think extra white space on large monitors is less of a problem than content cutting off on laptops. I am attaching an image of what the cut off looks like vs the scrollable content.
Screenshot 2020-08-05 at 9 52 52 AM
Screenshot 2020-08-05 at 9 51 51 AM

File: search-sentiments-map/public/scripts/modal-script.js (hunk 8 above)
Comment (carmenBen) [reply search-sentiments-map/public/scripts/modal-script.js]: I like the idea. My only concern is that we right now don't have coordinates for a country to center it with. So I'm not sure what I would pass as the latlng to center the map.

File: search-sentiments-map/public/scripts/map-script.js (hunk 2 above)
Comment (carmenBen) [reply search-sentiments-map/public/scripts/map-script.js:147]: Changed it to follow Haylee's suggestion. If we change our mind on this let me know.

File: search-sentiments-map/public/style.css (hunk 9 above)
Comment (ChenyuHeidiZhang) [reply search-sentiments-map/public/style.css:146]: Is the first the content cut that you talked about? I think I put a bottom padding to make it look less squeezed. But yeah if the screen is too small to display the entire panel then maybe we should go with the scroll

File: search-sentiments-map/public/scripts/modal-script.js (hunk 8 above)
Comment (ChenyuHeidiZhang) [reply search-sentiments-map/public/scripts/modal-script.js]: There's definitely data on this online, but yeah only if you have extra time

File: search-sentiments-map/public/style.css (hunk 9 above)
Comment (carmenBen) [reply search-sentiments-map/public/style.css:146]: Yeah if the screen is too small(like on a laptop), the content just cuts off.

File: search-sentiments-map/public/scripts/modal-script.js (hunk 8 above)
Comment (carmenBen) [reply search-sentiments-map/public/scripts/modal-script.js]: You think that this feature is worth adding a whole other json file?

Comment (carmenBen) [reply search-sentiments-map/public/scripts/modal-script.js]: If we move forward with this, it can be in a separate PR.

**Pull request merged**